index.html

```diff
@@ -2,10 +2,37 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Lab 06</title>
+    <title>Salmon Cookie Sales</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Condensed' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-
+  <div class=container>
+    <div class="jumbotron">
+      <div class="jumbotron-inner"></div>
+      <h1 class="header">Seattle Cookies</h1>
+    </div>
+    <section>
+      <h1 class="title">Cookies Sold Per Hour</h1>
+    </section>
+    <div class="hightlight">
+    <form id = "new-location">
```

Comment:
@kaylynyuh multiple areas still contain a space between the property, equal sign, and value on your HTML elements